Library/Homebrew/extend/ARGV.rb

```diff
@@ -268,6 +268,10 @@ def fetch_head?
     include? "--fetch-HEAD"
   end
 
+  def ignore_installed_specs?
+    include? "--ignore-installed-specs"
```

Comment:
Then build `godep` from source by default if a `devel` is spec is installed if we know (not speculate) that in the majority of cases this is known to be broken using a `devel` go with stable `go`. If we do that we should also probably ensure users fall back to `stable` as soon as there's a newer `stable` than the `devel` when they first `brew install --devel`d.

Alternatively if we don't actually know this is broken: just always use whatever version is installed (`stable`, `devel` or `head`) for dependencies and not care until we get user reported breakages and then figure out a solution for those cases. I don't think speculative bug fixing is useful.